package.json

```diff
@@ -16,29 +16,24 @@
     "test": "tests"
   },
   "scripts": {
-    "build": "tsc",
     "lint": "npm-run-all --aggregate-output --continue-on-error --parallel 'lint:!(fix)'",
     "lint:fix": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*:fix",
     "lint:hbs": "ember-template-lint .",
     "lint:hbs:fix": "ember-template-lint . --fix",
     "lint:js": "eslint . --cache",
     "lint:js:fix": "eslint . --fix",
-    "prepack": "npm run build",
     "start": "ember serve",
     "test": "npm-run-all lint test:*",
-    "test:jest": "jest",
     "test:ember": "ember test",
     "test:ember-compatibility": "ember try:each"
   },
   "dependencies": {
-    "babel-import-util": "^0.2.0",
+    "@babel/plugin-transform-class-static-block": "^7.22.5",
+    "babel-plugin-ember-template-compilation": "^2.2.0",
```

Comment:
These don't belong here.

Reply:
thanks!

Reply:
it seems that ember-cli-babel is not automatically adding `@babel/plugin-proposal-class-static-block` now

Reply:
looks like the lockfile's browserslist was way out of date :sweat_smile: 

```
Latest version:     1.0.30001538
Installed versions: 1.0.30001425, 1.0.30001538
```